packages/mgt/src/components/mgt-teams-channel-picker/mgt-teams-channel-picker.scss

```diff
@@ -7,28 +7,10 @@
 
 @import '../../../../../node_modules/office-ui-fabric-core/dist/sass/References';
 @import '../../styles/shared-styles.scss';
+@import '../../styles/theme.scss';
 
 $accent-color: var(--accent-color, #0078d4);
 
-$font-color: var(--font-color, #323130);
-$placeholder-default-color: var(--placeholder-default-color, #605e5c);
-$placeholder-focus-color: var(--placeholder-focus-color, #323130);
-$arrow-fill: var(--arrow-fill, #252424);
-
-$input-border: var(--input-border, 2px solid #b3b0ad);
-$input-border-bottom: var(--input-border-bottom, $input-border);
-$input-border-right: var(--input-border-right, $input-border);
-$input-border-left: var(--input-border-left, $input-border);
-$input-border-top: var(--input-border-top, $input-border);
-
-$input-hover-color: var(--input-hover-color, #323130);
-$input-focus-color: var(--input-focus-color, #0078d4);
-$input-background-color: var(--input-background-color, white);
-
-$dropdown-item-hover-background: var(--dropdown-item-hover-background, rgba(241, 241, 241, 0.87));
-$dropdown-background-color: var(--dropdown-background-color, white);
-$dropdown-item-selected-background: var(--dropdown-item-selected-background, #deecf9);
-
 :host {
   font-family: var(--default-font-family, 'Segoe UI');
```

Comment:
Same here, re globally defined vars

Reply:
Hmmm these should be defined in the theme.css file... Not sure why you are seeing them.

Reply:
sorry, I meant the `--default-font-family` var - should we have a spot to define these global vars in one place and reuse them in all components? It might be something we'd do in another PR, just thought I'd ask anyway.

Reply:
That's a great suggestion! I will make a issue and move that to a different PR.